modules/core/example/ts/create-policy.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Create a velocity limit policy on a multi-sig wallet at BitGo.
+ *
+ * This tool will help you see how to use the BitGo API to easily
+ * create new wallet policies
+ *
+ * Copyright 2019, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo, Wallet } from '../../src';
+const bitgo = new BitGo({ env: 'test' });
+
+// TODO: set your access token here
+const accessToken = '';
+
+// TODO: put the new policy on the wallet with this id
+const id = '';
+
+const coin = 'tltc';
+
+async function main() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+
```

Comment:
If ```id``` is empty string, I'd print a console message saying "make sure you add your ID to the script"

Reply:
I'd rather not add this to every example - it's extra boilerplate that doesn't help to understand how to use the API. This is not meant to be production code.

Reply:
ok fair enough. how about just adding one line to the README.md before ```cd example/ts``` that says something like "update the example script with the required TODO's" ?

Reply:
Good idea, will update